docs/source/features/ray.rst

```diff
@@ -0,0 +1,372 @@
+This guide assumes that there is a Ray cluster already running, and that this script is run locally on the cluster, or
+that the cluster job submission address is known.
+
+1.) Testing that the cluster works can be done as follows.
+
+.. code-block:: bash
+
+  ./isaaclab.sh -p source/standalone/workflows/ray/wrap_isaac_ray_resources.py --test
+
+2.) Submitting resource-wrapped sub-jobs can be done as described in the following file:
+
+.. dropdown:: source/standalone/workflows/ray/wrap_isaac_ray_resources.py
+  :icon: code
+
+  .. literalinclude:: ../../../source/standalone/workflows/ray/wrap_isaac_ray_resources.py
+    :language: python
+    :emphasize-lines: 12-64
+
+3.) For tuning jobs, specify the hyperparameter sweep similar to the following two files.
+
+.. dropdown:: source/standalone/workflows/ray/hyperparameter_tuning/rl_games_vision_cfg.py
+  :icon: code
+
+  .. literalinclude:: ../../../source/standalone/workflows/ray/hyperparameter_tuning/vision_cartpole_cfg.py
+    :language: python
+    :emphasize-lines: 17-35
+
+.. dropdown:: source/standalone/workflows/ray/hyperparameter_tuning/vision_cartpole_cfg.py
+  :icon: code
+
+  .. literalinclude:: ../../../source/standalone/workflows/ray/hyperparameter_tuning/vision_cartpole_cfg.py
+    :language: python
+    :emphasize-lines: 17-35
+
+Then, see the local examples in the following file to see how to start a tuning run.
+
+.. dropdown:: source/standalone/workflows/ray/isaac_ray_tune.py
+  :icon: code
+
+  .. literalinclude:: ../../../source/standalone/workflows/ray/isaac_ray_tune.py
+    :language: python
+    :emphasize-lines: 17-35
+
+
+
+To view the logs, simply run ``tensorboard --logdir=<LOCAL_STORAGE_PATH_READ_FROM_OUTPUT>``
+
+Remote Ray Cluster Setup and Use
+'''''''''''''''''''''''''''''''''
+This guide assumes that one desires to create a cluster on a remote host or server. This
+guide includes shared steps, and KubeRay or Ray specific steps. Follow all shared steps (part I and II), and then
+only the KubeRay or Ray steps depending on your desired configuration, in order of shared steps part I, then
+the configuration specific steps, then shared steps part II.
+
+Shared Steps Between KubeRay and Pure Ray Part I
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1.) Build the Isaac Ray image, and upload it to your container registry of choice.
+
+.. code-block:: bash
+
+  # Login with NGC (nvcr.io) registry first, see docker steps in repo.
+  ./isaaclab.sh -p docker/container.py start
+  # Build the special Isaac Lab Ray Image
+  docker build -t <REGISTRY/IMAGE_NAME> -f source/standalone/workflows/ray/cluster_configs/Dockerfile .
+  # Push the image to your registry of choice.
+  docker push <REGISTRY/IMAGE_NAME>
+
+KubeRay Specific
+~~~~~~~~~~~~~~~~
+`k9s <https://github.com/derailed/k9s>`_ is a great tool for monitoring your clusters that can
+easily be installed with ``snap install k9s --devmode``.
+
+1.) Verify cluster access, and that the correct operators are installed.
+
```

Comment:
For some reason this line doesn't show up as bold

Reply:
it was bc of indent oops